sql/create.go

```diff
@@ -151,8 +151,27 @@ func (p *planner) CreateTable(n *parser.CreateTable) (planNode, *roachpb.Error)
 		return nil, pErr
 	}
 
-	if pErr := p.createDescriptor(tableKey{dbDesc.ID, n.Table.Table()}, &desc, n.IfNotExists); pErr != nil {
+	created, pErr := p.createDescriptor(tableKey{dbDesc.ID, n.Table.Table()}, &desc, n.IfNotExists)
+	if pErr != nil {
 		return nil, pErr
 	}
+
+	if created {
```

Comment:
The case where the table already exists is a failed operation - to me, but does return false.